### ISSUE TITLE
Keep a list of speakers to connect with organizers

### ISSUE THREAD

**Issue by badboy** [rust-community/switchboard-team#7]
The main task of the switchboard team is to connect people, but we generally don't have an easy way to keep track of people we know and could reach out to.

Previously we tried to start a list of potential speakers we can reference when asked, but never fully finished or used that list.

**Comment by Manishearth** [comment]
Yeah, we should do this. Do you want to take point? We can also bring it up at tomorrow's all-team meeting.

**Comment by Deedasmi** [comment]
One thing I've noticed is that over on rust-www the "Friend request" issue form asks people for their organization contact, but as far as I can doesn't actually record it anywhere except in the closed issues. Has any work been done to centralize these addresses and verify if they are still correct? Or has there been much work in reaching out and seeing if these contacts would be willing to speak at events?

**Comment by Manishearth** [comment]
I think the production outreach group will know more of this cc @jonathandturner
